docs/dev-process/pull-request-process.md

```diff
@@ -0,0 +1,51 @@
+# Pull Request Process
+
+Requirements for a pull request to be accepted into PowerShell
+* Writing tests
+* Writing documentation
+
+## Pull Request Workflow
+
+1. The PR *author* creates a pull request from a fork.
+1. The *author* ensures that their pull request passes the [CI system][ci-system] build.
+  - If the build fails, a [Repository Maintainer][repository-maintainer] adds the `Review - waiting on author` label to the pull request. 
```

Comment:
there should be 3 spaces before `-` so that it could be indented correctly.
This comment applies to all instances where sub bullets exist in a list entry.
I know some of these problems have been there before this change, but it's better to fix them to make the doc more readable.

Reply:
Currently we don't use the labels. 😕

Reply:
Yes, strictly applying correct labels means the maintainer needs to closely monitor the status changes of the PR, e.g. has any reviewer finished reviewing and waiting on the author to address comments? has the author addressed comments and ready to have reviewers take another look? was the CI failed? I think that's not a good use of the maintainer's time.

Most of those status changes are obvious to all parties. Since author should drive the PR, generally the author should ping the reviewers once comments are addressed.

Reply:
will fix